docs/antrea-l7-network-policy.md

```diff
@@ -329,24 +329,24 @@ Allow ingress from client (10.10.1.9) to web (10.10.1.10/public/*).
 }
 ```
 
-Deny ingress from client (10.10.1.9) to web (10.10.1.10/admin/*)
+Deny ingress from client (10.10.1.4) to web (10.10.1.3/admin/*).
 
 ```json
 {
-  "timestamp": "2024-08-26T22:38:26.019956+0000",
-  "flow_id": 642636870504569,
+  "timestamp": "2024-09-04T21:00:09.613227+0000",
+  "flow_id": 1179717331488573,
   "in_iface": "antrea-l7-tap0",
   "event_type": "alert",
   "vlan": [
-    2
+    1
   ],
-  "src_ip": "10.10.1.9",
-  "src_port": 37892,
-  "dest_ip": "10.10.1.10",
+  "src_ip": "10.10.1.4",
+  "src_port": 35396,
+  "dest_ip": "10.10.1.3",
   "dest_port": 80,
   "proto": "TCP",
-  "pkt_src": "wire/pcap",
-  "tenant_id": 2,
+  "pkt_src": "stream (flow timeout)",
```

Comment:
If the pkt src is `stream (flow timeout)`, you are probably looking at the wrong message in the logs. There should be a message with src `wire/pcap` that shows up immediately, instead of after a flow timeout (which takes 30s to a minute IIRC)